### PR TITLE
feat: noticket - Only install required parts of AWS SDK

### PR DESCRIPTION
**Pull request by AaronKalair**

### CODE REVIEW

File: test/fixtures/cookbooks/test-jlaws/recipes/default.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 include_recipe 'jlaws'
-Chef::Resource::Log.send(:include, Jlaws::Helper)
+Chef::Resource::Log.send(:include, Helper)
```
Comment (AaronKalair) [inline test/fixtures/cookbooks/test-jlaws/recipes/default.rb]: I don't understand what this is doing but it does still function correctly

```
Recipe: test-jlaws::default
  * log[Node i-015745ebceae7ab40 (leeroy) is running!] action write

  * log[Node i-015745ebceae7ab40 is terminated!] action write (skipped due to only_if)
  * log[Node i-1111111 is not logged as it does not exist!] action write (skipped due to only_if)
```

File: .kitchen.yml [hunk 2]
```diff
@@ -1,19 +1,37 @@
...
+  name: dokken
 
 provisioner:
-  name: chef_zero
+  name: dokken
 
 platforms:
-  - name: ubuntu-14.04
-  - name: ubuntu-16.04
   - name: ubuntu-18.04
```
Comment (jameslegg) [inline .kitchen.yml]: want to also test this on aws linux?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
+### 2.0.0
+  - Only install required parts of AWS SDK
+  - Upgrade to support Chef Client 14
```
Comment (jameslegg) [inline CHANGELOG.md]: mention that we are dropping support for 12.x ?

File: .kitchen.yml (hunk 2 above)
Comment (stefano-pogliani) [reply .kitchen.yml]: Would be nice actually

**Comment by AaronKalair** [comment]
> looks, good, can you update the readme that still talks about Jlaws::Helper as well please?

Done!

**Pull request merged**